app/surveys/mci/mci_0205.py

```diff
@@ -0,0 +1,353 @@
+
+s1.add_question(q1)
+questionnaire.register(q1)
+
+q1r1 = Response()
+q1r1.id = "6fd644b0-798e-4a58-a393-a438b32fe637"
+q1r1.code = "11"
+q1r1.label = "From"
+q1r1.guidance = ""
+q1r1.type = "Date"
+q1r1.options = []
+q1r1.mandatory = True
+q1r1.messages = {
+    "MANDATORY": "Please answer before continuing.",
+    "INVALID_DATE": "The date entered is not valid.  Please correct your answer."
+}
+
+q1.add_response(q1r1)
+questionnaire.register(q1r1)
+
+q1r2 = Response()
+q1r2.id = "06a6a4b7-6ce4-4687-879d-3443cd8e2ff0"
+q1r2.code = "12"
+q1r2.label = "To"
+q1r2.guidance = ""
+q1r2.type = "Date"
+q1r2.options = []
+q1r2.mandatory = True
+q1r2.messages = {
+    "MANDATORY": "Please answer before continuing.",
+    "INVALID_DATE": "The date entered is not valid.  Please correct your answer."
+}
+
+q1.add_response(q1r2)
+questionnaire.register(q1r2)
+
+###
+# Section Two
+###
+
+s2 = Section()
+s2.id = "d9d25a21-41b8-42b2-ac7b-fb21b1036d71"
+s2.title = "Commodities - Retail Turnover"
+s2.description = "<p>- You should enter figures for the reporting period stated above<br>- You should round your figures to the nearest (£) pound</p><h4>Include</h4><ul><li>VAT</li><li>Internet Sales</li></ul>"  # NOQA
+
+block.add_section(s2)
+questionnaire.register(s2)
+
+q2 = Question()
+q2.id = "eedb9f9f-2f2e-41f4-9186-ba7110d9e6a4"
+q2.title = ""
+q2.description = ""
+q2.type = "Currency"
+
+s2.add_question(q2)
+questionnaire.register(q2)
+
+q2r = Response()
+q2r.id = "bb8168e6-2272-450d-b5a7-d3170508efb2"
+q2r.code = "22"
+q2r.label = "What was the value of the business's total sales of food?"
+q2r.guidance = "<div> <h4>Include</h4> <ul> <li>all fresh food</li> <li>other food for human consumption (except chocolate and sugar confectionery)</li> <li>soft drinks</li> </ul> </div> <div> <h4>Exclude</h4> <ul> <li>sales from catering facilities used by customers</li> </ul> </div>"  # NOQA
+q2r.type = "Currency"
+q2r.options = []
+q2r.mandatory = False
+q2r.messages = {
+    "NOT_INTEGER": "Please only enter whole numbers into the field.",
+    "NEGATIVE_INTEGER": "The value cannot be negative. Please correct your answer.",
+    "INTEGER_TOO_LARGE": "The maximum value allowed is 9999999999. Please correct your answer."
+}
+
+q2.add_response(q2r)
+questionnaire.register(q2r)
+
+q3 = Question()
+q3.id = "5743fe04-e9a3-491f-b475-ea5be662eff3"
+q3.title = ""
+q3.description = ""
+q3.type = "Currency"
+
+s2.add_question(q3)
+questionnaire.register(q3)
+
+q3r = Response()
+q3r.id = "fee0b9fe-4c3a-4c14-9611-4fa9e2e9578a"
+q3r.code = "23"
+q3r.label = "What was the value of the business's total sales of alcohol, confectionery and tobacco?"
+q3r.guidance = "<div> <h4>Include</h4> <ul> <li>alcoholic drink</li> <li>chocolate and sugar confectionery</li> <li>tobacco and smokers’ requisites</li></ul> </div>"  # NOQA
+q3r.type = "Currency"
+q3r.options = []
+q3r.mandatory = False
+q3r.messages = {
+    "NOT_INTEGER": "Please only enter whole numbers into the field.",
```

Comment:
Are these keys stored anywhere, I mean "NOT_INTEGER" that could be used instead (enum?) instead of pasting in the strings directly?